migrations/automatic/educandu-2021-12-16-01-delete-namespaces.js

```diff
@@ -0,0 +1,23 @@
+// eslint-disable-next-line camelcase
+export default class Educandu_2021_12_16_01_delete_namspaces {
+  constructor(db) {
+    this.db = db;
+  }
+
+  async up() {
+    await this.db.collection('documents').dropIndex('_idx_namespace_slug_');
+    await this.db.collection('documents').createIndexes([
+      {
+        name: '_idx_slug_',
+        key: { slug: 1 }
```

Comment:
This should actually only live in the elmu db...

Reply:
True, we have to discuss how we wanne do such things